earthpy/tests/test_stack.py

```diff
@@ -18,6 +18,45 @@ def in_paths(basic_image_tif):
     return [basic_image_tif] * 4
 
 
+@pytest.fixture
+def in_paths_mismatch(basic_image_tif, basic_image_tif_2):
+    """ Input file paths for tifs of different sizes to stack. """
+    return [basic_image_tif] * 4 + [basic_image_tif_2]
+
+
+@pytest.fixture
+def in_paths_CRS_mismatch(basic_image_tif, basic_image_tif_CRS):
```

Comment:
SHould this be 4 images with different CRS' to stack?

Reply:
@nkorinek just a note that you may be able to use these fixtures in your crop_all tests.

Reply:
@lwasser I don't think we need 4 different CRS, just 2 different ones at a minimum. The check is for the `set` of CRS present in the source list to be == 1. Having 2 CRS here supports that test.